Sources/NIOSSH/SSHClientConfiguration.swift

```diff
@@ -23,11 +23,22 @@ public struct SSHClientConfiguration {
     /// The global request delegate to be used with this client.
     public var globalRequestDelegate: GlobalRequestDelegate
 
+    /// Supported data encryption algorithms
+    public var transportProtectionSchemes: [NIOSSHTransportProtection.Type]
```

Comment:
I'm suggesting to put all configurable cryptographic algorithms in `SSHClientConfiguration` and `SSHServerConfiguration`, next candidate could be `keyExchangeAlgorithms`. And all static members will become `bundled*`. Wdyt? cc @Joannis

Reply:
Works for me.